diff --git a/node_modules/@sendbird/chat/lib/__definition.d.ts b/node_modules/@sendbird/chat/lib/__definition.d.ts
index d9c8b95..766d7d6 100644
--- a/node_modules/@sendbird/chat/lib/__definition.d.ts
+++ b/node_modules/@sendbird/chat/lib/__definition.d.ts
@@ -1,4 +1,3 @@
-import type AsyncStorage from '@react-native-async-storage/async-storage';
 
 export declare class AdminMessage extends BaseMessage {
   message: string;
@@ -1177,7 +1176,7 @@ export declare interface SendbirdChatParams<Modules extends Module[]> {
   debugMode?: boolean;
   localCacheEnabled?: boolean;
   localCacheEncryption?: Encryption;
-  useAsyncStorageStore?: typeof AsyncStorage;
+  useAsyncStorageStore?: any;
 }
 
 export declare class SendbirdError extends Error {
